docs/deploying_an_application.md

```diff
@@ -0,0 +1,109 @@
+# Empire :: Deploying an Application
+
+Before using Empire successfully it's important to understand it's design and architecture. It's suggested you first read the `Features & Design Goals document`.
+
+There are a few key things which your applications, and overall architecture, will need to keep in mind in order to be successful with Empire.  Below is an exhaustive set of criteria your applications must meet in order to work with Empire.  It's not that long and by no means locks you in to Empire.  Structing your application in this way provides many benefits, only one of which is being able to be managed by Empire.
+
+For the impatient, there are two absolutes:
+
+- Your must create a `Procfile` which defines how to run your application
+- If you application answers requests over the network, it must listen on port 8080, or better, on the `$PORT` environment variable.
```

Comment:
Might be worth mentioning that we really, really suggest you use $PORT if you can.  The reason being that it will allow us to change it later without breaking the app.  I don't see a good reason why we might change it, but it'd be cool to be able to.